lib/landrush/action/redirect_dns.rb

```diff
@@ -24,7 +24,7 @@ def redirect_dns
       def _target_host
         case provider
         when :virtualbox then
-          '10.0.2.2'
```

Comment:
It's good that you ask, I'm not sure 100% about this change!

Docs:
> In NAT mode, the guest network interface is assigned to the IPv4 range 10.0.x.0/24 by default where x corresponds to the instance of the NAT interface +2. So x is 2 when there is only one NAT instance active. In that case the guest is assigned to the address 10.0.2.15, the gateway is set to 10.0.2.2 and the name server can be found at 10.0.2.3.
https://www.virtualbox.org/manual/ch09.html#changenat

So I *think* `10.0.2.3` is correct. But we can also leave `10.0.2.2` as nobody complained so far, and it worked just fine?